testfiles/Fault_HumidistatOffset_Supermarket.idf

```diff
@@ -2431,8 +2431,10 @@
 	
   FaultModel:HumidistatOffset, 
     Humi_Offset_SalesFloor, !- Name
+    SalesFloor Humidistat,  !- Humidistat Name
+    ThermostatOffsetIndependent, !- Humidistat Offset Type
     ,  				  		!- Availability Schedule Name
     , 				  		!- Severity Schedule Name
     5,        	 	  		!- Reference Humidistat Offset
```

Comment:
@rongpengzhang Lines like this should have the units in the end-line comment
5,     !- Reference Humidistat Offset {percent}
Likewise for the thermostat offsets in other test files.
@Myoldmopar LKL used to run the example files through transition (I think) in a mode that "prettied" them up and fixed things like this automatically. Have you ever done that?